### DISCUSSION TITLE
Only one way get SP-API developer account ? pay USD 39.99 get seller account

### DISCUSSION THREAD

**Discussion by linroex** [amzn/selling-partner-api-models#3147] [General] [answered]
Hi

I am confuse, I was a developer, I want write some app for seller use, but read docs, the only way I get develop SP-API permission is "become a seller" first !? and then I need pay 39.99 per month? 

It's confuse for me, USD 39.99 are more than Apple Developer Account

Is Amazon has developer account can registe? like shopify dev account.

thanks!

**Comment by teddy-codes** [comment, marked as answer]
Amazon doesn't currently have any way to make a developer account. All accounts are sellercentral accounts. Of course, this is less than ideal, but this is what we have.

When you make a sellercentral account, you will have literally no sales, but you are able to register/manage your app. With MWS, I can understand why this is a requirement, but with the sp-api, it seems like they could have something better than this as a solution.

They probably just have little motivation to change it since they recently added all the app store related things into `manage apps` section of seller central.

Also, as a side note: Amazon is huge, so little change takes a while to happen. This is why we are just now seeing a JSON API and the introduction of oauth at Amazon. Developer accounts are ages away if going to exist at all.

**Reply by linroex** [reply]
thank you very much, yes I can understand, developer account is a big featur
